CONTRIBUTING.md

```diff
@@ -278,6 +278,19 @@ Below is a checklist of things to be mindful of when implementing a new instrume
 - Testing
   - When adding a new instrumentation remember to update `tox.ini` adding appropriate rules in `envlist`, `command_pre` and `commands` sections
 
+### Update supported instrumentation package versions
+
+- Update the respective instrumentation `pyproject.toml` file in _instruments_ entry under `[project.optional-dependencies]` section
+- Update instrumentation `package.py` file in `_instruments` variable
+- Run command `tox -e generate` to regenerate necessary files
+
+If you're adding support for a new version of the instrumentation package, follow these additional steps:
+
+- Add new test-requirements.txt file with the respective package version required for testing
```

Comment:
similarly, I think it helps to clarify the first step is in the package directory and the second from the repository root.